example/example_resourcemanager_test.go

```diff
@@ -0,0 +1,35 @@
+package example
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/oracle/oci-go-sdk/common"
+	"github.com/oracle/oci-go-sdk/example/helpers"
+	"github.com/oracle/oci-go-sdk/resourcemanager"
+)
+
+func ExampleResourceManagerListStacks() {
+	client, err := resourcemanager.NewResourceManagerClientWithConfigurationProvider(common.DefaultConfigProvider())
```

Comment:
Thanks for adding the samples!

Please add more complete samples before merge it, such as create, update stack or jobs. thanks!

Reply:
Sure.

Reply:
it's not necessary to put real data here, you can add comments and link to rest API doc about it. It could be something like 'base64 encoded terraform template. details can be found here: link'